### PR TITLE
Bridge Constants Refactor Integration

### PR DESCRIPTION
**Pull request by wilmerrootstock**
## Description
All these changes are part of a general Bridge Constants Refactor:

- Moving BridgeConstants class and all classes (regtest, testnet, mainnet, devnet) related to co.rsk.peg.constants package to Peg package.
- Moving the BridgeConstantsTest class to the same package inside the test section.
- For the sake of decoupling RSKj, we are applying some changes to remove the dependency between BridgeConstants and Federation classes.
- The FederationMember class contains a KeyType enum. This enum has a Value field that must be declared with the final keyword.
- releaseBtc method is printing a message showing the wrong unit. We changed the log message to print Weis as the unit instead of Satoshis getting the value in Weis from the Rsk transaction before converting it to satoshis with rskTx.getValue().
- There are 2 bridge constants related to the minimum pegout value. legacyMinimumPegoutTxValueInSatoshis and minimumPegoutTxValueInSatoshis. These names are not precise, because the data type is actually a Coin object, so it’s not an amount and should not have a unit. For this reason, we renamed both constants to legacyMinimumPegoutTxValue and minimumPegoutTxValue respectively.

*fed:bridge-constants-refactor-integration*

## Motivation and Context

- We are passing all the objects to Peg package to put all objects related to Powpeg together. With this, we are taking a step forward to decouple Powpeg.
- Decouple RSKj
- This is part of a tech debt and improves good practices on the project. The Final keyword will make this attribute immutable, which is a good practice for handling concurrent transactions and avoiding state modification after creation.
- This improves the traceability of the project.
- To improve naming conventions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Comment by wilmerrootstock** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

File: rskj-core/src/main/java/co/rsk/peg/constants/BridgeTestNetConstants.java [hunk 1]
```diff
@@ -51,23 +48,14 @@ public class BridgeTestNetConstants extends BridgeConstants {
...
-        // Change upon implementation of the <INSERT FORK NAME HERE> fork.
-        List<FederationMember> federationMembers = FederationMember.getFederationMembersFromKeys(genesisFederationPublicKeys);
-
-        // Currently set to:
-        // Currently set to: Monday, October 8, 2018 12:00:00 AM GMT-03:00
-        Instant genesisFederationAddressCreatedAt = Instant.ofEpochMilli(1538967600l);
-
-        FederationArgs federationArgs = new FederationArgs(federationMembers, genesisFederationAddressCreatedAt, 1L, getBtcParams());
-        genesisFederation = FederationFactory.buildStandardMultiSigFederation(federationArgs);
+        genesisFederationCreationTime = ZonedDateTime.parse("2018-10-08T03:00:00Z").toInstant();
```
Comment (nathanieliov) [inline rskj-core/src/main/java/co/rsk/peg/constants/BridgeTestNetConstants.java:58]: Is the instant object created from `ZonedDateTime` class the same as `Instant.ofEpochMilli(1538967600l)`?

Comment (wilmerrootstock) [reply rskj-core/src/main/java/co/rsk/peg/constants/BridgeTestNetConstants.java:58]: Yes, you can see that in `BridgeConstantsTest` class. In there, we make sure that this is the correct value.

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Review by marcos-iov — APPROVED** [review]

**Comment by marcos-iov** [comment]
pipeline:run

**Pull request merged**